.github/workflows/ci.yaml

```
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         php:
-          - '8.0'
+          - '7.4'
```

It doesn't mean the lowest supported version, we tend to use at least `8.1` for `4.6` branch:
```suggestion
          - '8.1'
```